app-antivirus/fangfrisch/files/fangfrisch-has-news.sh

```diff
@@ -38,12 +38,13 @@ usage() {
 
 gen_header() {
```

Comment:
For future reference, you shouldn't change files like this in-place (as they'll affect the older versions too). I guess it's fine here as the change is harmless but let's avoid it in future. Thanks!